examples/crm/src/Header.tsx

```diff
@@ -7,14 +7,15 @@ import { UserMenu, Logout, LoadingIndicator } from 'react-admin';
 const PREFIX = 'Header';
 
 const classes = {
-    root: `${PREFIX}-root`,
     logo: `${PREFIX}-logo`,
 };
 
-const Root = styled('nav')({
-    [`&.${classes.root}`]: {
-        flexGrow: 1,
-    },
+const Root = styled('nav', {
```

Comment:
It's probably unecessary in the CRM to support themes overrides. A simple mui `Box` with proper props (`component` and `sx`) should be enough. Same for the logo